config/routes.rb

```diff
@@ -558,14 +558,17 @@ def scoped_navigation_routes(context, navigation)
         resources :submission_pdfs, only: [:show], module: 'state_file/questions', path: 'questions/submission_pdfs'
         resources :federal_dependents, only: [:index, :new, :create, :edit, :update, :destroy], module: 'state_file/questions', path: 'questions/federal_dependents'
         resources :unemployment, only: [:index, :new, :create, :edit, :update, :destroy], module: 'state_file/questions', path: 'questions/unemployment'
+        get "/faq", to: "state_file/faq#index", as: :state_faq
+        get "/faq/:section_key", to: "state_file/faq#show", as: :state_faq_section
+        get "/data-import-failed", to: "state_file/state_file_pages#data_import_failed"
+      end
+
+      scope ':us_state', constraints: { us_state: /az|ny|us/ } do
```

Comment:
This will make the param case insensitive which may reduce the amount of 404's if for w/e reason the state param isn't the correct case
```suggestion
      scope ':us_state', constraints: { us_state: /az|ny|us/i } do
```